packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -35,14 +35,8 @@ export class OrgList implements vscode.Disposable {
     this.statusBarItem.show();
   }
 
-  public async displayDefaultUsername() {
-    let defaultUsernameorAlias: string | undefined;
-    if (hasRootWorkspace()) {
-      defaultUsernameorAlias = await OrgAuthInfo.getDefaultUsernameOrAlias(
-        false
-      );
-    }
-    if (defaultUsernameorAlias) {
+  public async displayDefaultUsername(defaultUsernameorAlias?: string) {
```

Comment:
This doesn't need to be `async` anymore.

Reply:
Ah good catch